.github/workflows/release.yaml

```diff
@@ -10,13 +10,11 @@ defaults:
     shell: bash
 
 env:
-  DOCKER_REPO: lightninglabs
```

Comment:
We no longer need this?

Reply:
Cleared up offline: these env variables were never even used in the first place!